src/placeos-driver/proxy/driver.cr

```diff
@@ -104,8 +104,14 @@ class PlaceOS::Driver::Proxy::Driver
         if error = result.error
           backtrace = result.backtrace || [] of String
           exception = PlaceOS::Driver::RemoteException.new(result.payload, error, backtrace)
-          @system.logger.warn(exception: exception) { exception.message }
-          raise exception
+          runtime_error = RuntimeError.new "remote exception", exception
+
+          begin
+            raise runtime_error
+          rescue error_with_backtrace
+            @system.logger.warn(exception: error_with_backtrace) { exception.message }
+            raise error_with_backtrace
+          end
```

Comment:
Any reason you can't do this?
```suggestion
          runtime_error = RuntimeError.new "remote exception", exception
          @system.logger.warn(exception: error_with_backtrace) { exception.message }
          raise runtime_error
```

Reply:
Isn't that what's essentially happening already?

Reply:
as there is no backtrace unless you raise the error